src/sentry/api/endpoints/organization_config_repositories.py

```diff
@@ -8,15 +8,25 @@
 
 
 class OrganizationConfigRepositoriesEndpoint(OrganizationEndpoint):
+    def has_access(self, organization, provider_id):
+        if provider_id == 'bitbucket' and not features.has(
```

Comment:
Could we possibly just do something like provider.has_access on the abstraction itself, and let plugin decide how thats tuned (e.g. it can implement the feature flag).

Alternatively we can make it a generic flag (like ``features.has('organizations:repo-provider', provider=provider_id)``).